srml/staking/src/phragmen.rs

```diff
@@ -16,43 +16,35 @@
 
 //! Rust implementation of the Phragmén election algorithm.
 
-use rstd::prelude::*;
-use primitives::PerU128;
-use primitives::traits::{Zero, Saturating, Convert};
-use parity_codec::{HasCompact, Encode, Decode};
-use crate::{Exposure, BalanceOf, Trait, ValidatorPrefs, IndividualExposure};
+use rstd::{prelude::*, collections::btree_map::BTreeMap};
+use primitives::{PerU128};
+use primitives::traits::{Zero, Convert, Saturating};
+use parity_codec::{Encode, Decode};
+use crate::{BalanceOf, Assignment, RawAssignment, ExpoMap, Trait, ValidatorPrefs};
 
 type Fraction = PerU128;
-type ExtendedBalance = u128;
-const SCALE_FACTOR_64: u128 = u64::max_value() as u128 + 1;
-
-/// Configure the behavior of the Phragmen election.
-/// Might be deprecated.
-pub struct ElectionConfig<Balance: HasCompact> {
-	/// Perform equalize?.
-	pub equalize: bool,
-	/// Number of equalize iterations.
-	pub iterations: usize,
-	/// Tolerance of max change per equalize iteration.
-	pub tolerance: Balance,
-}
+pub type ExtendedBalance = u128;
```

Comment:
If this is `pub`/used elsewhere, should there be some kind of warning, like "must be u64 or greater" (and any other restrictions) since it is used below with `u32::max_value() as ExtendedBalance + 1`?

Reply:
Good point. Yet, this is not something configurable. In fact it is set-in-stone since the module is asking for an explicit `Convert<Balance, u64>`, so this will always be equal to u128. 

And, this is really just a bit of syntax sugar otherwise the intention is to _always_ use u128.